<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreScorePage</class>
 <widget class="QMainWindow" name="PreScorePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>767</x>
      <y>879</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>461</width>
      <height>811</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>461</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="title">
        <string>评分界面</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>你当前选择的分数为：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="scoreEdit">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>36</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>50</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="minimumSize">
           <size>
            <width>411</width>
            <height>274</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="topMargin">
                <number>15</number>
               </property>
               <property name="bottomMargin">
                <number>15</number>
               </property>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">	background-color: rgb(229, 229, 229);</string>
                 </property>
                 <property name="text">
                  <string>很好</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">	background-color: rgb(229, 229, 229);</string>
                 </property>
                 <property name="text">
                  <string>好</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">	background-color: rgb(229, 229, 229);</string>
                 </property>
                 <property name="text">
                  <string>一般</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">	background-color: rgb(229, 229, 229);</string>
                 </property>
                 <property name="text">
                  <string>差</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">	background-color: rgb(229, 229, 229);</string>
                 </property>
                 <property name="text">
                  <string>很差</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="scoreSlider">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QSlider {
	padding-top: 15px;
	padding-bottom: 15px;
	border-radius: 5px;
}
 
QSlider::add-page:vertical {
	background-color: rgb(85, 170, 255);
	width:5px;
	border-radius: 2px;
}
 
QSlider::sub-page:vertical {
	
	background-color: rgb(229, 229, 229);
	width:5px;
	border-radius: 2px;
}
 
QSlider::groove:vertical {
	background:transparent;
	width:80px;
}
 
QSlider::handle:vertical {
	height: 20px;
	width: 20px;
	margin: 0px -4px 0px -4px;
	border-radius: 10px;
	background: rgb(255, 255, 255);
}</string>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="singleStep">
                <number>1</number>
               </property>
               <property name="pageStep">
                <number>2</number>
               </property>
               <property name="value">
                <number>50</number>
               </property>
               <property name="sliderPosition">
                <number>50</number>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::NoTicks</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="title">
        <string>进度显示</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_13">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>当前评价图片：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="finishedEdit">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_14">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>当前所评价图片区域：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="currentPartEdit">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>160</y>
      <width>271</width>
      <height>548</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>271</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>140</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>评分提示：</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>43</y>
       <width>251</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);
border:none;</string>
     </property>
     <property name="plainText">
      <string>    可以看出，刚才显示的照片的质量很不错，因此您可以去拖动左边的滑块，选择一个较高的分数，
    例如：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>388</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>点击“确认评分”之后</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>414</y>
       <width>220</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>即可开始下一张图的评测</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>240</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>每幅图片在评价完整体分数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>466</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>后需要评价局部分数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>492</y>
       <width>240</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>局部感知和整体感知会存在</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>518</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>差异</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>230</y>
       <width>113</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>38</pointsize>
      </font>
     </property>
     <property name="text">
      <string>90分</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnSure">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>410</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认评分</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
